controllers/errorHandlers.controllers.js

```diff
@@ -0,0 +1,22 @@
+exports.handlePSQL400s = (err, req, resp, next) => {
```

Comment:
Though I appreciate you thinking ahead. It's best practise to only include code which applies to this ticket in a PR
This ticket does not require the use of most of these error handling middleware functions, so 
`handlePSQL400s`, and `handleCustomErrors` should be removed for this PR